Maple2.File.Ingest/Utils/PyParameter.cs

```diff
@@ -134,6 +137,18 @@ internal record PyParameter(ScriptType Type, string Name) {
                 "None" => "Weather.Clear",
                 _ => $"Weather.{TriggerTranslate.ToPascalCase(value)}",
             },
+            ScriptType.EnumBannerType => string.IsNullOrWhiteSpace(value) ? null : value switch {
+                // The type values intentionally do not match the BannerType enum values.
+                //"0" => "",
+                "1" => "BannerType.Text",
+                //"2" => "",
+                "3" => "BannerType.Winner",
```

Comment:
Winner/Loser or Win/Lose feels more consistent

Reply:
if it's to match text that's fine too

Reply:
yeah the banners actually say "Winner" and "Lose"